boards/esp32-ttgo-t-beam/include/periph_conf.h

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (C) 2019 Yegor Yefremov
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_esp32_ttgo-t-beam
+ * @brief       Peripheral MCU configuration for TTGO T-Beam board
+ * @{
+ *
+ * TTGO T-Beam is an ESP32 development board with 4 MB Flash that
+ * uses the EPS32 chip directly. It integrates a SemTech SX1276 or SX1278 for
+ * LoRaWAN communication in the 433 MHz or the 868/915 MHz band, respectively.
+ * Additionally, it has an GPS receiver connected via UART on board.
+ *
+ * For detailed information about the configuration of ESP32 boards, see
+ * section \ref esp32_comm_periph "Common Peripherals".
+ *
+ * @note
+ * Most definitions can be overridden by an \ref esp32_app_spec_conf
+ * "application-specific board configuration".
+ *
+ * @file
+ * @author      Yegor Yefremov <yegorslists@googlemail.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+/**
+ * @name    ADC and DAC channel configuration
+ * @{
+ */
+
+/**
+ * @brief   Declaration of GPIOs that can be used as ADC channels
+ *
+ * @note As long as the GPIOs listed in ADC_GPIOS are not initialized as ADC
+ * channels with the \ref adc_init function, they can be used for other
+ * purposes.
+ */
+#ifndef ADC_GPIOS
+#define ADC_GPIOS   { GPIO36, GPIO39, GPIO32, GPIO33, \
```

Comment:
Is there any reason for the order, just the pinout? Especially GPIO36 and GPIO39 are not the first choice for ADC channels since they have connected capacitors.

Reply:
I have reordered the GPIOs and removed GPIO36 and GPIO39.